content/docs/command-reference/remote/add.md

```diff
@@ -350,13 +350,14 @@ $ dvc remote add myremote https://example.com/path/to/dir
 
 <details>
 
-### Click for WebDav
+### Click for WebDAV
 
 ```dvc
-$ dvc remote add myremote webdavs://example.com/path/to/dir
+$ dvc remote add myremote webdav://example.com/path/to/dir
```

Comment:
Oops I guess it was `webdavs` per https://github.com/iterative/dvc/pull/3647/files